tools/pipelines/templates/include-set-package-version.yml

```diff
@@ -12,6 +12,17 @@ parameters:
 
 # Set version
 steps:
+- task: Bash@3
+  name: InstallBuildTools
+  displayName: Install Fluid Build Tools
+  inputs:
+    targetType: 'inline'
+    workingDirectory: ${{ parameters.buildDirectory }}
+    script: |
+      npm link "$(Build.SourcesDirectory)/tools/build-tools"
+      cd "$(Build.SourcesDirectory)/tools/build-tools"
+      npm run build
```

Comment:
don't you need `npm i` in the build-tools directory?

Reply:
In testing it ran install automatically, but I agree explicit is better, and will let me use `npm ci` instead.

Reply:
Interesting,  I didn't know about that behavior.  If that is a documented behavior,  we can skip the npm install to save time

Reply:
I can't find clear documentation on this behavior, so I have updated the script to run `npm ci` first, then build, then link.